src/Command.js

```diff
@@ -62,6 +62,14 @@ export const builder = {
     describe: "Set max-buffer(bytes) for Command execution",
     type: "number",
     requiresArg: true
+  },
+  "json": {
+    describe: dedent`
```

Comment:
Let's move the `json` option descriptor into `LsCommand` and `UpdatedCommand` directly. It's an acceptable amount of duplication to maintain the benefit of targeted, relevant `--help` output. The description could be shorter, like "Show information in JSON format". (when in doubt, i copy npm's [docs](https://docs.npmjs.com/cli/ls#json))